### PR TITLE
Disable zoom cursor for expand/collapse on inline diff comments in PRs (#127)

### LINKED ISSUE #127: Comment context diffs have non-working zoom cursor for collapsing
When you are looking at diff comment in a PR, e.g. https://github.com/sindresorhus/refined-github/pull/112#discussion-diff-57938519 you still have the zoom cursor on the file headers for collapsing, but it's not working.

I guess we could try to remove them from there instead of supporting collapsing as I don't see them much needed in the inline diffs.

### PR DESCRIPTION
**Pull request by DrewML**
Fixes #127.

### CODE REVIEW

File: extension/content.css [hunk 1]
```diff
@@ -196,7 +196,7 @@
 }
 
 /* indicate collapsible file headers with zoom cursor */
-.file-header[data-path], .file-header[data-path] .diffstat {
+.js-details-container .file-header[data-path], .file-header[data-path] .diffstat {
```
Comment (hkdobrev) [inline extension/content.css]: @DrewML This still does the same for me in the inline diff in #112.

However this works for me:

``` css
.js-details-container:not(.discussion-item) .file-header[data-path], .file-header[data-path] .diffstat {
	cursor: zoom-out !important;
}
.refined-github-minimized:not(.discussion-item) .file-header[data-path], .refined-github-minimized .file-header[data-path] .diffstat {
	cursor: zoom-in !important;
}
```

Comment (DrewML) [inline extension/content.css]: Great catch! Update pushed

File: extension/content.css [hunk 2]
```diff
@@ -196,11 +196,13 @@
...
-	cursor: zoom-out !important;
+.js-details-container:not(.discussion-item) .file-header[data-path],
+.file-header[data-path] .diffstat {
+    cursor: zoom-out !important;
 }
-.refined-github-minimized .file-header[data-path], .refined-github-minimized .file-header[data-path] .diffstat {
-	cursor: zoom-in !important;
+.refined-github-minimized:not(.discussion-item) .file-header[data-path],
+.refined-github-minimized .file-header[data-path] .diffstat {
+    cursor: zoom-in !important;
```
Comment (hkdobrev) [inline extension/content.css]: I've overlooked the diffstat thing in my previous comment though. I think we would need the `:not()` thing for the `.diffstat` selector as well. What do you think?

Comment (DrewML) [inline extension/content.css]: Hmm, I don't *think* I've seen the diffstats at the comment-level diff, but I could be wrong. Have you?

Comment (hkdobrev) [inline extension/content.css]: @DrewML You are completely right! Haven't thought of that, I was just thinking abstractly.

**Comment by hkdobrev** [comment]
LGTM :mag:

**Comment by paulmolluzzo** [comment]
:+1: LGTM  

@DrewML Side note: Since you're a collaborator now can't you push directly to this repo? Saves the trouble of maintaining a fork.

**Comment by sindresorhus** [comment]
@paulmolluzzo @hkdobrev @DrewML Just merge PRs when they've gotten at least one `LGTM`. You don't have to wait for me ;)

**Comment by hkdobrev** [comment]
@sindresorhus :+1: I'm used to the GitHub Flow where the person who has opened the PR merges it after it has received a LGTM. This way the person who is most knowledgeable about the state of the PR and whether he needs to add something else is in control of merging it. http://githubflow.github.io/

**Comment by sindresorhus** [comment]
> This way the person who is most knowledgeable about the state of the PR and whether he needs to add something else is in control of merging it.

But if you add more stuff after the review you introduced unreviewed changes, which might have issues. I think if you plan to add more you should say so and ask for another review before it's merged.

**Comment by paulmolluzzo** [comment]
:+1: this should also ensure there are less conflicts since master won't have to have a bunch of things merged simultaneously.

If there's a LGTM written already the next person can just do the merge. I don't think the original author should need to merge, that'd mean the second person would write LGTM and we'd still wait on someone (the original author) to push a button. What's the point?

I agree with @sindresorhus that the merge shouldn't introduce anything more than the PR has without review. And the PR should be up to date to allow a clean merge into master.

@sindresorhus what about release tagging? I think we should leave that for you to do when you actually release to the webstore. It's probable that some times multiple merges will happen before a release is actually made, and that's OK.

**Comment by sindresorhus** [comment]
> what about release tagging?

Yeah. It takes me like 10 sec to tag a release.

**Pull request merged**